training/general/src/environments/lightgbm-3.2-ubuntu18.04-py37-cpu/context/conda_dependencies.yaml

```diff
@@ -0,0 +1,25 @@
+name: lightgbm-3.2-ubuntu18.04-py37-cpu
+channels:
+- anaconda
+- conda-forge
+dependencies:
+- python=3.8
+- pip=21.3.1
+- pandas>=1.1,<1.2
+- scipy~=1.5.0
+- numpy~=1.21.0
+- pip:
+  - matplotlib>=3.3,<3.4
+  - psutil>=5.8,<5.9
+  - tqdm>=4.59,<4.60
+  - ipykernel~=6.0
+  - azureml-core=={{latest-pypi-version}}
+  - azureml-defaults=={{latest-pypi-version}}
+  - azureml-mlflow=={{latest-pypi-version}}
+  - azureml-telemetry=={{latest-pypi-version}}
+  - scikit-learn~=0.24.1
+  - debugpy~=1.6.3
+  - xgboost~=1.4.0
+  - lightgbm~=3.2.0
```

Comment:
this is 2 years old, can we have a newer version?

Reply:
yes updated to lightgbm 3.3